app/subscriber/src/components/sidebar/advanced-search/styled/AdvancedSearch.tsx

```diff
@@ -107,4 +136,29 @@ export const AdvancedSearch = styled(Row)`
       background-color: rgb(103 80 164);
     }
   }
+
+  .date-range {
```

Comment:
if each section has the same `margin-top: 0.5em`, we should introduce another class or selector to apply the same `margin-top` to all sections

Reply:
Will do!